src/app/modules/filters/filters-content/filters-content.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit, EventEmitter, Output } from '@angular/core';
+
+@Component({
+  selector: 'ccf-filters-content',
+  templateUrl: './filters-content.component.html',
+  styleUrls: ['./filters-content.component.scss']
+})
+export class FiltersContentComponent implements OnInit {
+  filters: Record<string, unknown> = {};
+  @Output() applyFilters = new EventEmitter<Record<string, unknown>>();
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  updateTechnologies(event: string[]): void {
```

Comment:
This will add a bunch of redundant code, DRY by creating `updateFilter(key: string, value: unknown): void`